packages/core/core-flows/src/product/workflows/update-collections.ts

```diff
@@ -2,23 +2,33 @@ import { ProductTypes } from "@medusajs/types"
 import {
   WorkflowData,
   WorkflowResponse,
+  createHook,
   createWorkflow,
 } from "@medusajs/workflows-sdk"
 import { updateCollectionsStep } from "../steps"
 
 type UpdateCollectionsStepInput = {
   selector: ProductTypes.FilterableProductCollectionProps
   update: ProductTypes.UpdateProductCollectionDTO
+  additional_data?: Record<string, unknown>
 }
 
 type WorkflowInput = UpdateCollectionsStepInput
 
 export const updateCollectionsWorkflowId = "update-collections"
 export const updateCollectionsWorkflow = createWorkflow(
   updateCollectionsWorkflowId,
-  (
-    input: WorkflowData<WorkflowInput>
-  ): WorkflowResponse<ProductTypes.ProductCollectionDTO[]> => {
-    return new WorkflowResponse(updateCollectionsStep(input))
+  (input: WorkflowData<WorkflowInput>) => {
+    const updatedCollections = updateCollectionsStep(input)
+    const collectionsUpdated = createHook("collectionsUpdated", {
+      selector: input.selector,
+      update: input.update,
```

Comment:
thought: Don't think we need these 🤔

Reply:
Yeah @adrien2p pointed the same. Lemme remove them

Reply:
Fixed